src/main/java/com/philips/swcoe/cerberus/constants/DescriptionConstants.java

```diff
@@ -37,6 +37,8 @@
         "Specify minimum token length";
     public static final String LANGUAGE_OPTION_NOT_NULL_ARGUMENT_MESSAGE =
         "Specify language of the source code";
+    public static final String ALLOWED_LANGUAGES_EXCEPTION_MESSAGE =
+        "Unsupported language, Please specify 'java' for java language, 'cs' for csharp language and 'cpp' for C++";
     public static final String REPORT_STRUCTURE_OPTION_NOT_NULL_ARGUMENT_MESSAGE =
```

Comment:
To avoid dependency to update list of languages in multiple places, do you want to store a single list (for example via reflection on the enum you already use or an independent constant array or config data) and use this list to build the message string above and to build the regex (lang1|lang2|lang3)?

Reply:
Thanks @simaos , Thats a good thing to have. I will refactor this to be like that. So that new language addition will be of less debt

Reply:
I will create new PR for this refactor along with some test refactorings using Hamcrest .